io_u.h

```diff
@@ -30,6 +30,7 @@ enum {
  * The io unit
  */
 struct io_u {
+	uint8_t priority_bit;
```

Comment:
This packs horribly, you're added a 7 byte hole here. But this field after 'numberio' instead, then you fill a hole instead of creating a new one and growing the size of the io_u.